ipaserver/install/ipa_cacert_manage.py

```diff
@@ -112,6 +120,9 @@ def validate_options(self):
                 parser.error("certificate file name not provided")
         elif command == 'list':
             pass
+        elif command == 'delete':
+            if len(self.args) < 1:
+                parser.error("subject not provided")
```

Comment:
The error message should rather mention `nickame not provided` as the help mentions NICKNAME and not subject.